rllib/agents/ddpg/ddpg_torch_policy.py

```diff
@@ -163,9 +163,12 @@ def ddpg_actor_critic_loss(policy, model, _, train_batch):
 def make_ddpg_optimizers(policy, config):
     # Create separate optimizers for actor & critic losses.
     policy._actor_optimizer = torch.optim.Adam(
-        params=policy.model.policy_variables(), lr=config["actor_lr"])
+        params=policy.model.policy_variables(),
```

Comment:
Make Adam optimizer exactly the same in torch AND tf (torch default eps is 1e-8!).

Reply:
Compare with SAC (where we do this, too).